backend/typescript/services/implementations/authService.ts

```diff
@@ -44,6 +45,15 @@ class AuthService implements IAuthService {
       const googleUser = await FirebaseRestClient.signInWithGoogleOAuth(
         idToken,
       );
+
+      const userEmailValid = await validateUserEmail(googleUser.email);
```

Comment:
we can remove await since validateUserEmail isn't an async function anymore
```suggestion
      const userEmailValid = validateUserEmail(googleUser.email);
```